Library/Formula/xmlto.rb

```diff
@@ -40,10 +40,10 @@ def install
 +++ xmlto-0.0.25/xmlto.in
 @@ -209,7 +209,7 @@
  export VERBOSE
- 
+
  # Disable network entities
 -XSLTOPTS="$XSLTOPTS --nonet"
 +#XSLTOPTS="$XSLTOPTS --nonet"
- 
+
```

Comment:
Upon further investigation, the patch is not failing despite the changes in the file. But there seem to be other instances of hardcoded `nonet` options preventing `xmlto` from downloading the required files. See lines 545 and 548 [here](https://svn.fedorahosted.org/svn/xmlto/xmlto.in). This is just a guess so, because I am not sure about how xmlto, xmllint and xsltproc function.